### DISCUSSION TITLE
Duplicated timeseries in CollectorRegistry: {'xxxx'}

### DISCUSSION THREAD

**Discussion by cd06b9** [prometheus/client_python#735] [Q&A]
how to clear guage registry

my metrics format :  'serverid+ metricsname' , my serverid is auto add or del .

**Comment by cd06b9** [comment]
my mrtics name is variable, so i need clear this

**Comment by csmarchbanks** [comment]
Hello, to remove a metric from a collector you can use `registry.unregister(your_gauge)`. If you want to clear all of the metrics/label values on a single gauge, you can use `your_gauge.clear()`
